main/acle.md

```diff
@@ -4726,6 +4881,2404 @@ assertions, which may speed up access to aligned data (and will fault
+Intrinsics that explicitly read from the FFR do the second operation.
+First-faulting and non-faulting loads read from the FFRT and depend on
+its value, but they do not write to it.
+
+One consequence of this arrangement is that an intrinsic that writes to
+the FFR is only dead if there are no further references to the FFRT in
+the program. However, the normal “as if” rules apply, so the implementation
+can produce an empty code sequence for the write if doing so would not
+affect the behavior of the program. Similarly, if a first-faulting or
+non-faulting load follows an intrinsic that explicitly reads from the FFR,
+without an intervening write, the load keeps the read intrinsic alive
+even if the result of that read is unused. Again, the normal “as if”
+rules mean that the implementation can produce an empty code sequence
+for the read if doing so would not affect the behavior of the program.
+
+These FFRT dependencies are the only FFR-based ones that the implementation
+needs to consider when optimizing first-faulting and non-faulting loads;
+in all other respects the implementation can treat the loads like normal
+loads. This includes removing loads whose results are not used,
+suppressing loads of individual elements if their values do not matter,
+or reordering loads within a group (subject to the usual rules for
+normal loads). In practice the value of the FFR before a load does still
+affect which elements of the load result are defined, and in practice
+the loads do still write to the FFR, but the input program does not
+control these effects directly.
+
+Assuming `float64_t` data, the C version of the code above would be:
+
+``` c
+  svfloat64_t z0, z1, z2;
+  svbool_t p0, p1, p2, p3;
+  double *x0, *x1, *x2;
+  …
+  svsetffr();
+  z0 = svldff1[_f64](p0, x0);
+  z1 = svldff1[_f64](p1, x1);
+  z2 = svldff1[_f64](p2, x2);
+  p3 = svrdffr();
+```
+
+### List of SVE intrinsics
+
+The full list of SVE intrinsics can be found on
+[developer.arm.com](https://developer.arm.com/architectures/instruction-sets/intrinsics/#f:@navigationhierarchiessimdisa=[sve,sve2])
+This list includes SVE2 and other SVE extensions (such as the 32-bit
+matrix multiply extensions).
+
+The tick boxes on the left can be used to narrow the displayed
+intrinsics to particular data types and element sizes. Below that is an
+instruction group hierarchy that groups the intrinsics based on the
+operation that they perform.
+
+### Mapping of SVE instructions to intrinsics
+
+#### List of instructions
+
+This section contains a list of all FEAT_SVE instructions. For each one it
+gives a reference to the associated ACLE intrinsic or explains why no
+such intrinsic exists.
+
+| **Instruction**                      | **Intrinsic** |
+| ------------------------------------ | ------------- |
+| ABS                                  | [`svabs`](https://developer.arm.com/architectures/instruction-sets/intrinsics/#f:@navigationhierarchiesreturnbasetype=[int]&q=svabs) |
```

Comment:
Hmm, not sure.  I see the log also has:
```
2022-02-26T11:11:31.1678265Z [✖] https://developer.arm.com/architectures/instruction-sets/intrinsics/#q=svcntb_pat → Status: 403
```
which looks like a fairly innocuous uri.

Reply:
> [...] looks like a fairly innocuous uri.

Yeah, the server is refusing to load that too because it seeing all the other requests, so it doesn't matter what the rest of the URL looks like. After a certain threshold, all request are rejected.

Reply:
I tried fixing(?) the path in ci.yml and the latest version came back clean.  Would feel more confident in the result if it printed more than an “OK” line though :-)